kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/EventHandlerServlet.java

```diff
@@ -1,32 +1,43 @@
 /*******************************************************************************
- * Copyright (c) 2016 Eurotech and/or its affiliates and others
+ * Copyright (c) 2016, 2017 Eurotech and/or its affiliates and others
  *
  * All rights reserved. This program and the accompanying materials
  * are made available under the terms of the Eclipse Public License v1.0
  * which accompanies this distribution, and is available at
  * http://www.eclipse.org/legal/epl-v10.html
- * 
+ *
+ * Contributors:
+ *  Amit Kumar Mondal
```

Comment:
This is because I have written this code as an Eurotech intern and I have now finished my internship. Right now I am contributing as an individual. According to our previous discussion, the copyright header signifies that the contribution has been made from Eurotech as well as others. And contributors section is an optional element that can comprise the name of the contributors.